pygls/utils.py

```diff
@@ -0,0 +1,12 @@
+import asyncio
+
+import psutil
+
+
+async def process_watcher(pid, on_process_terminated, delay=5):
+    """Periodically checks if the give pid exists. Delay is in seconds.
```

Comment:
typos give -> given
Maybe also add the explanation of what `on_process_terminated` is - clarify that it is a function?